src/wp-content/themes/twentytwenty/assets/css/editor-style-block-rtl.css

```diff
@@ -352,19 +356,23 @@ Inter variable font. Usage:
 
 .editor-styles-wrapper p.has-small-font-size {
 	--wp--preset--font-size--small: 0.842em;
+	font-size: 0.842em;
 }
 
 .editor-styles-wrapper p.has-normal-font-size,
 .editor-styles-wrapper p.has-regular-font-size {
 	--wp--preset--font-size--normal: 1em;
+	font-size: 1em;
 }
 
 .editor-styles-wrapper p.has-medium-font-size {
 	--wp--preset--font-size--medium: 1.1em;
+	font-size: 1.1em;
 }
 
 .editor-styles-wrapper p.has-large-font-size {
 	--wp--preset--font-size--large: 1.25em;
+	font-size: 1.25em;
 }
 
 .editor-styles-wrapper p.has-larger-font-size {
```

Comment:
What about the "larger" font size? Should it receive `x-large` or similar preset?

Reply:
The existing issue is with the theme presets that have the same `slug` than one defined by WordPress. Because WordPress doesn't define any `larger` ([see](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/theme.json#L196)) this still works as before.

Reply:
Could it not use the `x-large` slug in this case? Or does there need to be a direction naming match been, i.e. the slug would need to be `larger` which isn't defined?

Reply:
This PR doesn't aim to make any design change, just adapting existing presets, so that change is not required. If people think that's something worth doing I'd address it as a separate issue where design is involved, I'm not the best person to respond to that question :)

Reply:
Isn't `x-large` is an existing preset as [defined here in theme.json]( https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/theme.json#L214)? If yes, couldn't that slug be used for the "larger" styles:

```css
.editor-styles-wrapper p.has-larger-font-size {
	--wp--preset--font-size--large: 1.5em;
	font-size: 1.5em;
}
```

(Sorry if my question is redundant. Seeking to understand)

Reply:
Yes, `x-large` is a new preset introduced by WordPress 5.9 and it'll generate the following class: `.has-x-large-font-size`. My point is that this PR only aims to fix how presets are updated by using CSS Custom Properties. So, as long as a theme doesn't have a `.has-x-large-font-size` class it doesn't need any change.

A different matter is whether the existing `.has-larger-font-size` provided by this theme can be `.has-x-large-font-size` now that WordPress has introduced the `x-large` preset. That's a design decision I don't have enough context about and that's not necessary for this PR. Does this help?

Reply:
Aha, I see. There is a direct correlation then between the slug and the CSS class generated. Cool. Thanks for your patience in explaining that to me.